src/views/Login.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div id="app">
```

Comment:
`id=app` is also used in bootstrap HTML, considering removing this id, use a class name if required.

Reply:
removed Id, replaced it with class.